docs/reference/content/guides/annotations.md

```diff
@@ -32,6 +32,16 @@ The [`@IndexOptions`]({{< apiref "org/mongodb/morphia/annotations/IndexOptions"
 definition.  This annotation replaces the fields found directly on the `@Index` annotation.  This annotation was added to ensure that index
 options are consistent across the various index definition approaches.
 
+#### Collation
+The [`@Collation`]({{< apiref "org/mongodb/morphia/annotations/Collation" >}}) annotation defines the options to apply to the collation of
+an index definition.  These collation definitions can significantly affect how documents are matched.  In addition to defining a collation
```

Comment:
Should we be more explicit than "can significantly affect how documents are matched?"  since it would only impact string comparisons?